app/views/onboarding/OnboardingTemplate.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {
+  ImageBackground,
+  StatusBar,
+  StyleSheet,
+  View,
+  ScrollView,
+  ImageSourcePropType,
+} from 'react-native';
+import { SvgXml } from 'react-native-svg';
+import { Button } from '../../components/Button';
+import { Typography } from '../../components/Typography';
+
+import { Colors, Spacing } from '../../styles';
+import { Theme } from '../../constants/themes';
+
+type OnboardingTemplateProps = {
+  iconXml: string;
+  title: string;
+  body: string;
+  primaryButtonLabel: string;
+  primaryButtonOnPress: () => void;
+  secondaryButtonLabel?: string;
+  secondaryButtonOnPress?: () => void;
+  theme: 'light' | 'dark';
+  background: ImageSourcePropType;
+  invertIcon?: boolean;
+};
+
+const OnboardingTemplate = ({
+  background,
+  iconXml,
+  title,
+  body,
+  primaryButtonLabel,
+  primaryButtonOnPress,
+  secondaryButtonLabel,
+  secondaryButtonOnPress,
+  invertIcon,
+  theme,
+}: OnboardingTemplateProps): JSX.Element => {
+  return (
+    <Theme use={theme === 'light' ? 'default' : 'violet'}>
```

Comment:
@johnschoeman I'm sorry for digging us deeper in this theming hole, this is only being used for light / dark themed buttons. I'd like to refactor the button component soon and remove these themes here & in the export template

Reply:
makes sense to me!